### PR TITLE
Update version information

### PR DESCRIPTION
**Pull request by fangyixiao18**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here and update the documentation.

## Checklist

**Before PR**:

- [ ] Pre-commit or other linting tools are used to fix the potential lint issues.
- [ ] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, like docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects, like MMDet or MMSeg.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,8 +119,8 @@ Supported algorithms:
 - [x] [SimSiam (CVPR'2021)](https://arxiv.org/abs/2011.10566)
 - [x] [Barlow Twins (ICML'2021)](https://arxiv.org/abs/2103.03230)
 - [x] [MoCo v3 (ICCV'2021)](https://arxiv.org/abs/2104.02057)
-- [x] [MAE](https://arxiv.org/abs/2111.06377)
-- [x] [SimMIM](https://arxiv.org/abs/2111.09886)
+- [x] [MAE (CVPR'2022)](https://arxiv.org/abs/2111.06377)
+- [x] [SimMIM (CVPR'2022)](https://arxiv.org/abs/2111.09886)
 - [x] [CAE](https://arxiv.org/abs/2202.03026)
```
Comment (tonysy) [inline README.md]:
```suggestion
- [x] [CAE](ArXiv'2022)(https://arxiv.org/abs/2202.03026)
```

File: README_zh-CN.md [hunk 2]
```diff
@@ -120,8 +119,8 @@ MMSelfSup 依赖 [PyTorch](https://pytorch.org/), [MMCV](https://github.com/open
 - [x] [SimSiam (CVPR'2021)](https://arxiv.org/abs/2011.10566)
 - [x] [Barlow Twins (ICML'2021)](https://arxiv.org/abs/2103.03230)
 - [x] [MoCo v3 (ICCV'2021)](https://arxiv.org/abs/2104.02057)
-- [x] [MAE](https://arxiv.org/abs/2111.06377)
-- [x] [SimMIM](https://arxiv.org/abs/2111.09886)
+- [x] [MAE (CVPR'2022)](https://arxiv.org/abs/2111.06377)
+- [x] [SimMIM (CVPR'2022)](https://arxiv.org/abs/2111.09886)
 - [x] [CAE](https://arxiv.org/abs/2202.03026)
```
Comment (tonysy) [inline README_zh-CN.md]:
```suggestion
- [x] [CAE](ArXiv'2022)(https://arxiv.org/abs/2202.03026)
```

File: docs/en/changelog.md [hunk 3]
```diff
@@ -2,6 +2,27 @@
...
+### v0.9.2 (28/07/2022)
+
+#### New Features
+
+- Support MAE Reconstructed Image Visualization ([#376](https://github.com/open-mmlab/mmselfsup/pull/376))
+
+#### Bug Fixes
+
+- Fix args/cfg bug in extract.py ([#357](https://github.com/open-mmlab/mmselfsup/pull/357))
+- Fix bugs of SimMIM config ([#360](https://github.com/open-mmlab/mmselfsup/pull/360))
```
Comment (YuanLiuuuuuu) [inline docs/en/changelog.md]: Better to be more concrete, this modification should be ``fix SimMIM mask generator config bug`` and change the content in Chinese version accordingly.

**Review by YuanLiuuuuuu — APPROVED** [review]

**Review by tonysy — APPROVED** [review]
LGTM

**Pull request merged**